gitclone/steperror.go

```diff
@@ -1,11 +1,178 @@
 package gitclone
 
-import "github.com/bitrise-io/bitrise-init/step"
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/bitrise-io/bitrise-init/step"
+)
+
+// DetailBuilder ...
+type DetailBuilder = func(...string) Detail
+
+// PatternErrorMatcher ...
+type PatternErrorMatcher struct {
+	defaultHandler DetailBuilder
+	handlers       map[string]DetailBuilder
+}
+
+func newPatternErrorMatcher(defaultHandler DetailBuilder, handlers map[string]DetailBuilder) *PatternErrorMatcher {
```

Comment:
calling a DetailBuilder as `handler` is a bit confusing, why not `defaultBuilder` and `patternToBuilder`?